doc/classes/PackedByteArray.xml

```diff
@@ -4,8 +4,10 @@
 		A packed array of bytes.
 	</brief_description>
 	<description>
-		An array specifically designed to hold bytes. Packs data tightly, so it saves memory for large array sizes.
+		An array specifically designed to hold bytes. It packs data tightly, so it saves memory for large array sizes.
 		[PackedByteArray] also provides methods to encode/decode various types to/from bytes. The way values are encoded is an implementation detail and shouldn't be relied upon when interacting with external apps.
+		[b]Note:[/b] PackedByteArray is a reference type. This means that changing the elements of an array will be reflected in other references to that array. To get a copy of an array that can be modified independently of the original array, use [method duplicate].
```

Comment:
This can be misleading, as when getting packed arrays from classes manipulating them won't change the original data as they are copy on write in c++, meaning that any change to one returned from a class in c++, or fed to one, won't affect it

Reply:
Does this apply to regular arrays as well? I put it here because within GDScript, it acts as a reference the same as Array and Dictionary.

Reply:
Nope, only Packed*Array, as it uses `Vector<T>` internally, it is relevant in the above case for example, but any time a native class returns a `Packed*Array` as a property it won't affect the original data when modified

Reply:
Would prefacing it with "In GDScript" be enough, then? Or should there be a mention about them being returned as copies by built-in classes?

Reply:
Unsure, I think something like "when returned from a built-in method or property", but will see if I can think of some clear wording tomorrow as I'm about to leave for the day